ffmpeg/transcoder.c

```diff
@@ -327,6 +329,11 @@ int transcode(struct transcode_thread *h,
     if(outputs[i].is_dnn_profile == 0) {
       ret = flush_outputs(ictx, &outputs[i]);
       if (ret < 0) LPMS_ERR(transcode_cleanup, "Unable to fully flush outputs")
+      //send EOF signal to signature filter
+      if(outputs[i].sfilters != NULL) {
+        av_buffersrc_close(outputs[i].sf.src_ctx, AV_NOPTS_VALUE, AV_BUFFERSRC_FLAG_PUSH);
+        free_filter(&outputs[i].sf);
+      }
```

Comment:
We should also ensure the filter is flushed and freed if any error occurs in our transcode loop and we hit `transcode_cleanup`. Maybe moving this whole section inside `transcode_cleanup:` makes more sense.